packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1258,6 +1331,40 @@ describe(SucceedRelationshipAttributeAndNotifyPeerUseCase.name, () => {
     });
 });
 
+describe(ChangeDefaultRepositoryAttributeUseCase.name, () => {
+    test("should change default repository attribute", async () => {
```

Comment:
Please write "RepositoryAttribute" instead of "repository attribute" here as you do in the tests above.